src/components/clicker.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+class Clicker extends React.Component {
+  constructor() {
```

Comment:
Although this is 100% correct, you don't need a constructor or a super anymore, and this.state can be defined just as state.